requirements.txt

```diff
@@ -1,5 +1,5 @@
 psutil>=5.7.0
-lightwood >= 0.27.1
+lightwood >= 0.30
```

Comment:
`0.29.0` and wait for the stable build to deploy before changing the version + switch tests to staging